src/components/availability-panel/drag-drop-context/DroppableComponent.tsx

```diff
@@ -18,13 +18,22 @@ const DroppableComponent: FC<droppableComponent> = ({
   unAssignedTasks,
   idleMembers,
   isTaskOnDrag,
+  searchTerm,
+  searchTermUnassignedTask,
 }) => (
   <div>
     {droppableId === 'tasks' ? (
       <Droppable droppableId="tasks" isCombineEnabled={!isTaskOnDrag}>
         {(provided) => (
           <div ref={provided.innerRef}>
-            {unAssignedTasks.map((task, index) => {
+            {unAssignedTasks.filter((val) => {
+              if (searchTermUnassignedTask === '') {
+                return true;
+              } if (val.title.toLowerCase().includes(searchTermUnassignedTask.toLowerCase())) {
```

Comment:
What does **val** mean here ?
Let's make it more meaningful such as **task** or **taskItem**